components/org.wso2.micro.integrator.initializer/src/main/java/org/wso2/micro/integrator/initializer/deployment/synapse/deployer/SynapseAppDeployer.java

```diff
@@ -1148,7 +1149,8 @@ public void deployArtifactType(List<Artifact.Dependency> artifacts, CarbonApplic
                 }
             }
 
-            JsonObject deployedArtifact = createUpdatedArtifactInfoObject(artifact, artifactPath);
+            String updateType = DEPLOY_UPDATE_TYPE;
```

Comment:
Do we need this line ? Can't we directly pass the type to the method?

Reply:
Fixed